### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by SadiqZak**

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    font-weight: 300;
+}
+
+.bd-400{
+    font-weight: 400;
+}
+
+.bd-500{
+    font-weight: 500;
+} */
```
Comment (romabulani) [inline CSS/style.css:43]: Delete code comments which are not in use

File: CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+}
+/* margin */
+.mg-t-xs{
+    margin-top: 10px;
+}
+
+/* background color */
+
+.bg-blue{
+    background-color: #4285F4;
```
Comment (romabulani) [inline CSS/style.css:93]: use variables for colors, dont hardcode

File: CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    cursor: pointer;
+}
+
+/* font size */
+
+/* .font-size{
+
+} */
+
+/* width */
```
Comment (romabulani) [inline CSS/style.css:134]: remove code not in use

File: CSS/style.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    height: 26px;
+}
+
+.ht-50{
+    height: 50px;
+}
+
+.ht-64{
+    height: 64px;
+}
```
Comment (romabulani) [inline CSS/style.css:184]: use rems for px > 8 or px > 10

File: Components/Alert/alert.html [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+                         <!-- component -->
+                         <div class="alert-outlined-warning ht-50"><p class="alert-p"><i class="fas fa-exclamation-circle alert-i"></i> This is a warning alert — check it out!</p></div>
+
+                          <!-- component -->
+                        <div class="alert-outlined-info ht-50"><p class="alert-p"><i class="fas fa-exclamation-circle alert-i"></i> This is an info alert — check it out!</p></div>
+
+                         <!-- component -->
+                         <div class="alert-outlined-success ht-50"><p class="alert-p"><i class="fas fa-exclamation-circle alert-i"></i> This is a success alert — check it out!</p></div>
+
```
Comment (romabulani) [inline Components/Alert/alert.html:64]: Give identation, use prettier

**Review by romabulani — COMMENTED** [review]
- Avoid using hardcoded colors, use variables 
- Use rems instead of px
- proper indentation
- remove code not in use instead of commenting it

**Review by varnita619 — APPROVED** [review]
looks good

**Pull request merged**